src/main/java/seedu/duke/Duke.java

```diff
@@ -44,11 +47,14 @@ protected static void runSlam() {
                 System.out.println(feedback.feedbackToUser);
                 Ui.printLineBreak();
             } catch (NullPointerException | NumberFormatException | StringIndexOutOfBoundsException e) {
-                System.out.println("Please key in the commands accurately, use help to view the guide");
+                Ui.printLineBreak();
+                System.out.println("Returning to Main Page...");
```

Comment:
This `catch` block for this exception is specific to the `update` command but the exceptions thrown might not be unique to that command. As such, would it be better instead to catch these exceptions somewhere within the `UpdateParser` class instead?